RELEASE.md

```diff
@@ -1,3 +1,32 @@
+## Release 0.0.15
+
+## New features 
+- Multimodal tensor combination (https://github.com/marqo-ai/marqo/pull/332, https://github.com/marqo-ai/marqo/pull/355). Combine image and text data into a single vector! Multimodal combination objects can be added as Marqo document fields. This can be used to encode text metadata into image vectors. See usage [here](https://docs.marqo.ai/0.0.15/Advanced-Usage/document_fields/#multimodal-combination-object).
+
+## Bug fixes
+- Fixed a bug that prevented CLIP's device check from behaving as expected (https://github.com/marqo-ai/marqo/pull/337)
+- CLIP utils is set to use the OpenCLIP default tokenizer so that long text inputs are truncated correctly (https://github.com/marqo-ai/marqo/pull/351). 
+
+## Contributor shout-outs:
+- Thank you to our 2.4k stargazers
+- Thank you to [@ed-muthiah](https://github.com/ed-muthiah), [@codebrain](https://github.com/codebrain) and others for raising issues.
+
+
+# Release 0.0.14
+
+## New features 
+- `use_existing_tensors` flag, for `add_documents` (https://github.com/marqo-ai/marqo/pull/335). Use existing Marqo tensors to autofill unchanged tensor fields, for existing documents. This lets you quickly add new metadata while minimising inference operations. See usage [here](https://docs.marqo.ai/0.0.14/API-Reference/documents/#query-parameters).
+- `image_download_headers` parameter for `search` and `add_documents` (https://github.com/marqo-ai/marqo/pull/336). Index and search non-publicly available images. Add image download auth information to `add_documents` and `search` requests. See usage [here](https://docs.marqo.ai/0.0.14/API-Reference/image_downloads/).
+
+## Optimisations
+- The index cache is now updated on intervals of 2 seconds (https://github.com/marqo-ai/marqo/pull/333), rather than on every search. This reduces the pressure on Marqo-OS, allowing for greater search and indexing throughput. 
+
+## Bug fixes
+- Helpful validation errors for invalid index settings (https://github.com/marqo-ai/marqo/pull/330). Helpful error messages allow for a smoother getting-started experience. 
+- Automatic conversion to `fp32` when using `fp16` models on CPU  (https://github.com/marqo-ai/marqo/pull/331). This allows Marqo running on a CPU-only machine to index and search a `fp16` CLIP index (although we still recommend interacting with a `fp16` index with `device=CUDA` for the best results). 
```

Comment:
"Automatic conversion to `fp32` when using `fp16` models on CPU  (https://github.com/marqo-ai/marqo/pull/331). This allows Marqo running on a CPU-only machine to index and search a `fp16` CLIP index (although we still recommend interacting with a `fp16` index with `device=CUDA` for the best results). " -> " Automatic precision conversion to `fp32` when using `fp16` models on CPU  (https://github.com/marqo-ai/marqo/pull/331)."